### ISSUE TITLE
Handling spaces in vehicle type

### ISSUE THREAD

**Issue by sprinkjm** [jmscslgroup/strym#46]
When parsing a vehicle name to infer the DBC type, it may be there is a space. this is not clearly handled. For example:

```
[2021_11_11_14_12_06] (root) INFO: Vehicle model infered is nissan-rogue sport-2020
```

When parsing a file like 2021-11-10-23-51-25_JN1BJ1CW3LW375199_CAN_Messages.csv

The challenge will come when looking for a dbc file named something like
```
[2021_11_11_14_14_28] (root) INFO: Vehicle model infered is nissan-rogue sport-2020
The dbcfile: /Users/sprinkle/opt/anaconda3/lib/python3.8/site-packages/strym/dbc/nissan_rogue sport_2020.dbc doesn't exist, or read permission error
```

**Comment by rahulbhadani** [comment]
Fetching sample data from Cyverse and Nissan DBC.

**Comment by rahulbhadani** [comment]
Fixed in commit fb4dac3
Pypi release:  0.4.12
